test/RpcCompatibility.js

```diff
@@ -243,6 +243,19 @@ describe('RpcCompatibility', () => {
             checkBlock(block, false);
             checkBlockTimeStamp(parseInt(block.timestamp, 16), 12000);
         });
+
+        it('blocks have valid parent hashes', async () => {
+            const withTransaction = false;
+            const block = await provider.send('eth_getBlockByNumber', ['latest', withTransaction]);
+            const parentBlock = await provider.send('eth_getBlockByHash', [block.parentHash, withTransaction]);
+
+            console.log('Block:', block);
+            console.log('Parent block:', parentBlock);
```

Comment:
No need to log unless debugging

Reply:
oops forgot to cleanup thanks! addressed as part of 75eeb36acf34872bc010efaf682d245903bb553d